src/go/rpk/pkg/cli/redpanda/admin/config/config.go

```diff
@@ -70,10 +74,81 @@ func newLogLevelCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	}
 	cmd.AddCommand(
 		newLogLevelSetCommand(fs, p),
+		newLogLevelGetCommand(fs, p),
 	)
 	return cmd
 }
 
+func newLogLevelGetCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var host string
+
+	cmd := &cobra.Command{
+		Use:   "get [LOGGERS...]",
+		Short: "Get broker logger's log level",
+		Long: `Get broker logger's log level.
+
+This command obtains the broker logger's log level. Each Redpanda
+broker has many loggers, and each can be individually returned.
+
+You can specify many loggers at once by separating each logger name with a space.
+To return all loggers, you can leave the logger argument empty or pass "all". 
+To see all possible loggers, run the following command:
```

Comment:
Not sure how to word it better, but `redpanda --help-loggers` only work where you have redpanda installed, so this will fail for Mac users.

Perhaps: `To see all possible loggers, run 'redpanda --help-loggers' in your node`

Or something along those lines